_meta/beat.reference.yml

```diff
@@ -78,6 +78,8 @@ apm-server:
   # or the IP and User Agent of the real user (frontend requests). It defaults to true.
   #capture_personal_data: true
 
+  #agents: ["nodejs", "python"]
```

Comment:
this defaults to "nodejs", "python", "ruby", "apm-js", "java", "go"
Let's add the rest of the agents here so that the config file reflects the defaults

Reply:
also in `_meta/beat.yml`,